### ISSUE TITLE
tridentctl install --generate-custom-yaml: Create setup directory if not exist

### ISSUE THREAD

**Issue by superbrothers** [NetApp/trident#307]
`tridentctl install --generate-custom-yaml` fails if `setup` directory does not exist:
```
$ tridentctl install --generate-custom-yaml
YAML generation failed; could not write namespace YAML file; open /workdir/setup/trident-namespace.yaml: no such file or directory
```

I prefer the behavior that creates `setup` directory if not exist. WDYT?

**Comment by adkerr** [comment]
This has already been fixed in 19.10 in 5441a38f60ec61f1b443ee9cc884d70477781eae

**Comment by superbrothers** [comment]
Great! Thank you.
